CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingPageSeenCounter.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  OnboardingPageSeenCounter.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import Foundation
+
+class OnboardingPageSeenCounter {
+    private var pages: [OnboardingPageModel]
+
+    init(pages: [OnboardingPageModel]) {
+        self.pages = pages
+    }
+
+    func pageNotSeen(_ page: OnboardingPageModel) -> Bool {
+        let initialPages = pages
```

Comment:
I can not do this since will modify directly the initial array `pages` and by doing this I can not use `markPageAsSeen` which is using `pages` array to filter out the page that was already seen.
I've added a comment to clarify this.

Reply:
@ValentinaIancu-Gini `contains` shouldn't mutate array

Reply:
I will check once again.